sdk/eventhub/azure-eventhubs/azure/eventhub/_consumer_client.py

```diff
@@ -0,0 +1,184 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import logging
+from typing import Any, Union, TYPE_CHECKING
+from ._common import EventPosition,\
+    EventHubSharedKeyCredential, EventHubSASTokenCredential
+from ._eventprocessor.event_processor import EventProcessor
+from ._client import EventHubClient
+if TYPE_CHECKING:
+    from azure.core.credentials import TokenCredential  # type: ignore
+
+log = logging.getLogger(__name__)
+
+
+class EventHubConsumerClient(EventHubClient):
+    """Represents an AMQP connection to an EventHub and receives event data from it.
+
+    Example:
+        .. literalinclude:: ../examples/test_examples_eventhub.py
+            :start-after: [START create_eventhub_consumer_client_sync]
+            :end-before: [END create_eventhub_consumer_client_sync]
+            :language: python
+            :dedent: 4
+            :caption: Create a new instance of the EventHubConsumerClient.
+    """
+
+    def __init__(self, host, event_hub_path, credential, **kwargs):
+        # type:(str, str, Union[EventHubSharedKeyCredential, EventHubSASTokenCredential, TokenCredential], Any) -> None
+        """
+        :param host: The hostname of the Event Hub.
+        :type host: str
+        :param event_hub_path: The path of the specific Event Hub to connect the client to.
+        :type event_hub_path: str
+        :param network_tracing: Whether to output network trace logs to the logger. Default
```

Comment:
We don't document this one yet (@johanste )

Reply:
changed to logging_enable